tests/test_scars.py

```diff
@@ -0,0 +1,632 @@
+"""Tests for the scars module."""
+from pathlib import Path
+import numpy as np
+import pytest
+
+from topostats import scars
+
+BASE_DIR = Path.cwd()
+RESOURCES = BASE_DIR / "tests" / "resources"
+
+
+def test_remove_scars(synthetic_scar_removal_args: dict):
+    """Test removal of scars"""
+
+    scars_removed, scar_mask = scars.remove_scars(**synthetic_scar_removal_args)
+
+    target_img = np.load(RESOURCES / "test_scars_synthetic_remove_scars.npy")
+    target_mask = np.load(RESOURCES / "test_scars_synthetic_remove_scars_mask.npy")
+
+    np.testing.assert_array_equal(scars_removed, target_img)
+    np.testing.assert_array_equal(scar_mask, target_mask)
+
+
+@pytest.mark.parametrize(
+    "potential_positive_scar, target, threshold_low, max_scar_width",
+    [
+        # Successful scar segment
+        (
+            np.array(
+                [
+                    [0],
+                    [3],
+                    [3],
+                    [3],
+                    [0],
+                ]
+            ),
+            np.array(
+                [
+                    [0],
+                    [3],
+                    [3],
+                    [3],
+                    [0],
+                ]
+            ),
+            2,
+            3,
+        ),
+        # Unsuccesssful scar segment: negative scar rather than positive
```

Comment:
:+1: Really useful for understanding what each set of parameters are doing, thanks. :smile: